### PR TITLE
Adding Isolated and Quarantined to BS

### PR DESCRIPTION
**Pull request by dominikgehl**

### CODE REVIEW

**Comment by andreasamsler** [comment]
BS is already doing this. Would the parser make your work easier, @jb3-2 ?

**Comment by jb3-2** [comment]
Yes it would make my life easier! Ideally the prser would also calculate the other BS specific columns to the right of source.

File: scrapers/scrape_bs.py [hunk 1]
```diff
@@ -125,4 +125,7 @@
     sc.find(r'Todesfälle\s*im\s*Kanton\s*Basel-Stadt\s*beträgt(?:\s*\S+)?\s*insgesamt\s*([0-9]+)\b', d) or \
     sc.find(r'Die\s*Zahl\s*der\s*Todesfälle\s*im\s*Kanton\s*Basel-Stadt\s*beträgt\s*.*unverändert\s*([0-9]+)\b', d)
 
+dd.isolated = sc.int_or_word(sc.find(r'\s+(\S+)\s+aktiven\s+Fällen\s+in\s+Isolation', d))
```
Comment (metaodi) [inline scrapers/scrape_bs.py:128]: @dominikgehl the text today is a bit different

> Die Zahl der 979 Infektionen setzt sich zusammen aus 923 genesenen Personen (-), 50 Todesfällen (-) und sechs aktiven Fällen (+ 1).

So for the `current_isolated` the part "in Isolation" is optional

Comment (dominikgehl) [reply scrapers/scrape_bs.py]: updated

**Comment by jb3-2** [comment]
Thank you very much!!

**Comment by metaodi** [comment]
@jb3-2 maybe we can get those other numbers as well. Currently the scraper scripts are not equipped to handle variables that only occur for one canton. Could you maybe create an issue for that?

**Comment by jb3-2** [comment]
@metaodi OK I have created an issue: https://github.com/openZH/covid_19/issues/796 - thanks! J.

**Pull request merged**